compiler/src/iree/compiler/Codegen/Common/GPU/GPUApplyTilingLevel.cpp

```diff
@@ -161,6 +165,23 @@ applyTileAndFuseToEachRoot(RewriterBase &rewriter,
     };
     tileAndFuseOptions.setFusionControlFn(controlFn);
 
+    RewritePatternSet cleanupPatterns(context);
+
+    {
+      // Add pattern to fuse pad operations without zero slice gaurd. In IREE,
+      // due to the way we tile, generally we do not have zero slices.
+      auto zeroSliceGaurd =
+          [](tensor::ExtractSliceOp sliceOp) -> std::optional<bool> {
```

Comment:
To avoid unused argument warnings
```suggestion
          [](tensor::ExtractSliceOp) -> std::optional<bool> {
```